app/services/usps_in_person_proofing/enrollment_helper.rb

```diff
@@ -62,13 +62,9 @@ def send_ready_to_verify_email(user, enrollment, is_enhanced_ipp:)
       # @return [String] The enrollment code
       # @raise [Exception::RequestEnrollException] Raised with a problem creating the enrollment
       def create_usps_enrollment(enrollment, pii, is_enhanced_ipp)
-        # Use the enrollment's unique_id value if it exists, otherwise use the deprecated
-        # #usps_unique_id value in order to remain backwards-compatible. LG-7024 will remove this
-        unique_id = enrollment.unique_id || enrollment.usps_unique_id
-
         applicant = UspsInPersonProofing::Applicant.new(
           {
-            unique_id: unique_id,
+            unique_id: enrollment.unique_id,
```

Comment:
Should `unique_id` be removed from the `UspsInPersonProofing::Applicant` as well?

Reply:
No, because we still need to send this field to USPS in our request to create a USPS enrollment.  The only thing we want to delete in this PR is the out of date method that was previously used to create the `unique_id` field for an enrollment.  The field itself is sticking around.